app/server/endpoint/src/main/java/io/syndesis/server/endpoint/v1/handler/integration/support/IntegrationSupportHandler.java

```diff
@@ -336,11 +332,10 @@ private  void importIntegrations(SecurityContext sec, JsonDbDao<Integration> exp
                 integrationHandler.update(id, builder.version(previous.getVersion()+1).build());
                 result.add(ChangeEvent.of("updated", integration.getKind().getModelName(), id));
             }
-            break;
```

Comment:
I think this `break` here is a bug that would prevent multiple integrations to load from an export. FYI @chirino @lburgazzoli

Reply:
do we have multiple integration in an export ?

Reply:
This seems to add multiple to the exported ZIP file:

https://github.com/syndesisio/syndesis/blob/c9760574058f36a8ecd7c8f4d81c39b9f7e32a53/app/server/endpoint/src/main/java/io/syndesis/server/endpoint/v1/handler/integration/support/IntegrationSupportHandler.java#L155-L162

Reply:
so maybe open a new issue :)